homeassistant/components/combined_energy/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "domain": "combined_energy",
+  "name": "Combined Energy",
+  "integration_type": "hub",
+  "documentation": "https://www.home-assistant.io/integrations/combined_energy",
+  "requirements": ["combined-energy-api==0.5"],
+  "codeowners": ["@timsavage"],
+  "iot_class": "cloud_polling",
+  "quality_scale": "silver",
+  "config_flow": true
```

Comment:
should be in domain, name, alphabetical order

Reply:
Is there documentation specifying the order? I'm not able to find any mention of it.

Order is irrelevant in JSON for object types. I'd be more inclined to order in the same order as the documentation lists the values.

Reply:
It isn't actually written down but it should be.

https://github.com/home-assistant/core/pull/87020

Run pre-commit and it should probably notify you

Reply:
Ah was added after I opened the PR.